### PR TITLE
add seclists pkgbuild

### PR DESCRIPTION
**Pull request by cerberusec**
add seclists pkgbuild

### CODE REVIEW

File: packages/seclists/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Maintainer heichblatt <git@hanneseichblatt.de>
+pkgname=seclists-git
+pkgbase=seclists
+pkgver=r283.8ef8694
+pkgrel=1
+pkgdesc="A collection of multiple types of lists used during security assessments."
```
Comment (Edu4rdSHL) [inline packages/seclists/PKGBUILD]: Use single quotes.

File: packages/seclists/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+arch=('any')
+url="https://github.com/danielmiessler/SecLists/"
```
Comment (Edu4rdSHL) [inline packages/seclists/PKGBUILD]: Use single quotes.

File: packages/seclists/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+groups=()
+depends=()
+makedepends=('git')
+provides=("${pkgname}")
+conflicts=()
+replaces=()
+backup=()
+options=()
+install=
+source=('seclists-git::git+https://github.com/danielmiessler/SecLists.git')
```
Comment (Edu4rdSHL) [inline packages/seclists/PKGBUILD]: Use $pkgname::source_url

File: packages/seclists/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+license=('unknown')
+groups=()
+depends=()
+makedepends=('git')
+provides=("${pkgname}")
+conflicts=()
+replaces=()
+backup=()
+options=()
+install=
```
Comment (Edu4rdSHL) [inline packages/seclists/PKGBUILD]: Remove all non-used options.

File: packages/seclists/PKGBUILD [hunk 5]
```diff
@@ -0,0 +1,33 @@
+# Maintainer heichblatt <git@hanneseichblatt.de>
+pkgname=seclists-git
+pkgbase=seclists
+pkgver=r283.8ef8694
+pkgrel=1
+pkgdesc="A collection of multiple types of lists used during security assessments."
+arch=('any')
+url="https://github.com/danielmiessler/SecLists/"
+license=('unknown')
+groups=()
```
Comment (Edu4rdSHL) [inline packages/seclists/PKGBUILD]: Add groups.

File: packages/seclists/PKGBUILD [hunk 6]
```diff
@@ -0,0 +1,33 @@
+# Maintainer heichblatt <git@hanneseichblatt.de>
+pkgname=seclists-git
```
Comment (Edu4rdSHL) [inline packages/seclists/PKGBUILD]: Remove -git and just use seclists.

File: packages/seclists/PKGBUILD [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+replaces=()
+backup=()
+options=()
+install=
+source=('seclists-git::git+https://github.com/danielmiessler/SecLists.git')
+noextract=()
+md5sums=('SKIP')
+
+pkgver() {
+    cd "$srcdir/${pkgname}"
```
Comment (Edu4rdSHL) [inline packages/seclists/PKGBUILD]: Use cd $pkgname

File: packages/seclists/PKGBUILD [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+    printf "r%s.%s" "$(git rev-list --count HEAD)" "$(git rev-parse --short HEAD)"
```
Comment (Edu4rdSHL) [inline packages/seclists/PKGBUILD]: Use the style specified in the PKGBUILD styles.

File: packages/seclists/PKGBUILD [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+
+package() {
+    mkdir -p "${pkgdir}/usr/share/wordlists/${pkgname}"
+    find "${srcdir}/${pkgname}" -maxdepth 1 | grep -v \\.git | xargs -I{} cp -r {} "${pkgdir}/usr/share/wordlists/${pkgname}"
```
Comment (Edu4rdSHL) [inline packages/seclists/PKGBUILD]: Just use cp -R

**Comment by noptrix** [comment]
thanks guys. @cerberusec please check the stuff from @Edu4rdSHL. at the end i personally will fix the groups. since this package is too big we have to exclude it from 'blackarch' and put it only in 'blackarch-exploitation' or so.

**Comment by cerberusec** [comment]
@Edu4rdSHL @noptrix thanks for the help, let me know if it needs anything else.

File: packages/seclists/PKGBUILD [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+pkgname=seclists
+pkgver=r283.8ef8694
+pkgrel=1
+pkgdesc='A collection of multiple types of lists used during security assessments.'
+arch=('any')
+url='https://github.com/danielmiessler/SecLists/'
+license=('unknown')
+groups=('blackarch-exploitation')
+makedepends=('git')
+provides=("${pkgname}")
```
Comment (Edu4rdSHL) [inline packages/seclists/PKGBUILD]: Remove it line.

File: packages/seclists/PKGBUILD [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+source=('$pkgname::source_url')
```
Comment (Edu4rdSHL) [inline packages/seclists/PKGBUILD]: source_url is anything here. When I said source_url it's `https://github.com/danielmiessler/$pkgname`

File: packages/seclists/PKGBUILD [hunk 12]
```diff
@@ -0,0 +1,25 @@
...
+sha512sums=('SKIP')
+
+pkgver() {
+    cd "${pkgname}"
```
Comment (Edu4rdSHL) [inline packages/seclists/PKGBUILD]: Just use `cd $pkgname`

File: packages/seclists/PKGBUILD [hunk 13]
```diff
@@ -0,0 +1,25 @@
...
+    echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+}
+
+
+package() {
```
Comment (Edu4rdSHL) [inline packages/seclists/PKGBUILD]: Use:
```
package() {
  cd $pkgname
  install -dm 755 "$pkgdir/usr/share/$pkgname"

  find -maxdepth 1 | grep -v \\.git | xargs -I{} cp -r {} "$pkgdir/usr/share/$pkgname"
}
```

File: packages/seclists/PKGBUILD (hunk 10 above)
Comment (cerberusec) [reply packages/seclists/PKGBUILD]: Which line? Did you mean "Remove **git** line."?

**Comment by noptrix** [comment]
@cerberusec thank you. btw, have a look to our changes (for next time).

**Pull request merged**